public/data/ben-dh3.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "Benjamin Duffield-Harding",
+  "type": "personal",
+  "bio": "Crypto, generative-art, and open source enthusiast",
+  "avatar": "https://github.com/ben-dh3.png",
+  "url": "https://linkfree.eddiehub.io/benjaminduffieldharding",
```

Comment:
AFAIK, this field should be removed. There's no `url` field in this part.
```suggestion
```


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Not maintainable

Reply:
Thanks for your help! :)